src/test/assembly/x86_64-fortanix-unknown-sgx-lvi-inline-assembly.rs

```diff
@@ -0,0 +1,41 @@
+// Test LVI load hardening on SGX inline assembly code
+
+// assembly-output: emit-asm
+// compile-flags: --crate-type staticlib
+// only-x86_64-fortanix-unknown-sgx
+
+#![feature(llvm_asm)]
+
+#[no_mangle]
+pub extern fn get(ptr: &u64) -> u64 {
+    let value : u64;
+    unsafe {
+        llvm_asm!(".start_inline_asm:
```

Comment:
New tests being added should probably be using just the new assembly syntax unless we can't for some reason, could you move this over to asm! instead?

Reply:
Moved it to `asm!`